quipucords/client/index.html

```diff
@@ -1,75 +1,84 @@
-              <div class="col-sm-10 col-md-10">
-                <input type="text" class="form-control" id="inputUsername" placeholder="" tabindex="1">
-              </div>
-            </div>
-            <div class="form-group">
-              <label for="inputPassword" class="col-sm-2 col-md-2 control-label">Password</label>
-              <div class="col-sm-10 col-md-10">
-                <input type="password" class="form-control" id="inputPassword" placeholder="" tabindex="2">
-              </div>
-            </div>
-            <div class="form-group">
-              <div class="col-xs-8 col-sm-offset-2 col-sm-6 col-md-offset-2 col-md-6">
-                <div class="checkbox">
-                  <label>
-                    <input type="checkbox" tabindex="3"> Remember username
-                  </label>
-                </div>
-                <span class="help-block"> Forgot <a href="#" tabindex="5">username</a> or <a href="#" tabindex="6">password</a>?</span>
-              </div>
-              <div class="col-xs-4 col-sm-4 col-md-4 submit">
-                <button type="submit" class="btn btn-primary btn-lg" tabindex="4">Log In</button>
-              </div>
-            </div>
-          </form>
-        </div>
+    <div class="nav-pf-vertical nav-pf-vertical-with-sub-menus nav-pf-persistent-secondary">
+      <ul class="list-group">
+        <li class="list-group-item">
+          <a href="javascript:void(0)">
+            <span class="fa fa-dashboard" data-toggle="tooltip" title="Sources"></span>
+            <span class="list-group-item-value">Sources</span>
+          </a>
+        </li>
+        <li class="list-group-item">
+          <a href="javascript:void(0)">
+            <span class="fa fa-shield" data-toggle="tooltip" title="Scans"></span>
+            <span class="list-group-item-value">Scans</span>
+          </a>
+        </li>
+        <li class="list-group-item">
+          <a href="javascript:void(0)">
+            <span class="fa fa-space-shuttle" data-toggle="tooltip" title="Credentials"></span>
+            <span class="list-group-item-value">Credentials</span>
+          </a>
+        </li>
+      </ul>
+    </div>
 
-        <div class="col-sm-5 col-md-6 col-lg-7 details">
-          <p>
-             <strong>Welcome to PatternFly!</strong>
-             This is placeholder text, only. Use this area to place any information or introductory
-              message about your application that may be relevant for users. For example, you might
-              include news or information about the latest release of your product here&mdash;such
-              as a version number.
-          </p>
-        </div>
+    <div class="container-fluid container-cards-pf container-pf-nav-pf-vertical nav-pf-persistent-secondary">
+      <div class="row">
+        Lorem ipsum content...
       </div>
     </div>
 
+    <script>
+      $(document).ready(function() {
+        $().setupVerticalNavigation(true);
```

Comment:
Why are we using the jquery vertical navigation?

Reply:
Same, placeholder bits. Copied from the Patternfly site.